### PR TITLE
Remove Duplicate AlternativeTo

### PR DESCRIPTION
**Pull request by gjhklfdsa**
AlternativeTo is listed twice under Tools.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2231,8 +2231,7 @@ <h3>Tools</h3>
...
     <li><a href="https://prism-break.org/"><strong>PRISM Break</strong></a> - We all have a right to privacy, which you can exercise today by encrypting your communications and ending your reliance on proprietary services.</li>
     <li><a href="https://securityinabox.org/"><strong>Security in-a-Box</strong></a> - A guide to digital security for activists and human rights defenders throughout the world.</li>
-    <li><a href="http://alternativeto.net/?license=opensource&platform=self-hosted&sort=likes"><strong>AlternativeTo.net</strong></a> - Great collection of open source online and self-hosted software sorted by likes.</li>
-    <li><a href="https://securedrop.org/"><strong>SecureDrop</strong></a> - An open-source whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. It was originally created
+=    <li><a href="https://securedrop.org/"><strong>SecureDrop</strong></a> - An open-source whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. It was originally created
```
Comment (Shifterovich) [inline index.html]: Why the `=`?

Comment (asddsaz) [reply index.html]: I'm still new to Github, what does += mean?

Comment (Shifterovich) [reply index.html]: It means that you accidentally added a `=` to the source code :) https://github.com/privacytoolsIO/privacytools.io/blob/c80880a7243b981576a4bab75ed1d6a03fd7eb3d/index.html#L2234

Comment (gjhklfdsa) [reply index.html]: Ahhh, that makes sense. It should be fixed now.

Sorry if I wasted you time. :)

**Comment by Shifterovich** [comment]
Thanks!

**Pull request merged**